riotctrl/shell/__init__.py

```diff
@@ -44,10 +44,14 @@ def __del__(self):
 
     def _start_replwrap(self):
         if self.replwrap is None or self.replwrap.child != self.riotctrl.term:
-            # consume potentially shown prompt to be on the same ground as if
-            # it is not shown
-            self.riotctrl.term.expect_exact([self.prompt, pexpect.TIMEOUT],
-                                            timeout=self.PROMPT_TIMEOUT)
+            # flush pexpect up to 10000 characters to fix syncing issues with
```

Comment:
```suggestion
            # flush pexpect up to 10000(arbitrarily large value) characters to start with an empty buffer.
            # This fixes issues where REPLWrapper would capture previous commands output.
```

Maybe the comment can be a little more detailed? Feel free to modify of course

Reply:
Done.